Bug Fixes

SimoneSchmidt opened the pull request:
- Fixed the way the WizardStepper is integrated in StartTaskModal and EditScheduledEventsModal, so that the navigation buttons on top work as intended.
 - Changed capture agent drop-downs to searchable drop-downs.
 - Changed time drop-downs in EventDetailsSchedulingTab to searchable drop-downs.

SimoneSchmidt committed:
Fixed top Navigation Buttons in Bulk Action Modals

Fixed the way the WizardStepper is integrated in StartTaskModal and EditScheduledEventsModal, so that the navigation buttons on top work as intended.

SimoneSchmidt committed:
Fixed small bug in event selection of EditScheduledEventsModal, that occurred, when changing the selection again, after having visited another tab.

SimoneSchmidt committed:
Changed capture agent drop-downs to searchable drop-downs.

isabellakutger commented on modules/admin-ui-picard/app/src/components/events/partials/ModalTabsAndPages/EditScheduledEventsEditPage.js:
```diff
@@ -137,7 +112,8 @@ const EditScheduledEventsEditPage = ({ previousPage, nextPage, formik, inputDevi
                                                                         </td>
                                                                         <td className="editable ng-isolated-scope">
-                                                                            <Field name={`editedEvents.${key}.changedTitle`}
+                                                                            <Field tabIndex={(key * 14) + 1}
```
leave here a comment, why you calculate the tab index this way

isabellakutger reviewed (changes requested):
- Search in the language field in wizards and modals is not working
- Fix style of dropdowns, currently all dropdowns have different styles (both searchable ones and non-searchable ones)
-

SimoneSchmidt committed:
New drop-down in EventDetailsSchedulingTab + Requested Changes

Changed time selection in Scheduling Tab of Event Details to new searchable drop-downs.
Small bug-fixes.
Changes according to review of PR #566.

SimoneSchmidt committed:
Removed some unused imports.

isabellakutger reviewed (changes requested):
- The dropdowns in the new event wizard for scheduling are still not searchable. The same goes for the edit scheduled events bulk modal.
- The style is better now, but in some places, there are still the old OC dropdowns and in some the plain HTML ones. Moreover, the searchable dropdowns should be a lot narrower to go with the style of the non-searchable ones

SimoneSchmidt commented:
> * The dropdowns in the new event wizard for scheduling are still not searchable. The same goes for the edit scheduled events bulk modal.
> 
>  * The style is better now, but in some places, there are still the old OC dropdowns and in some the plain HTML ones. Moreover, the searchable dropdowns should be a lot narrower to go with the style of the non-searchable ones

The remaining drop-downs will all be changed to the new drop-down component in the future.
